### PR TITLE
backport from `dav1d` 1.2.0: picture: allow storing an array of Dav1dITUTT35 entries

### PR DESCRIPTION
**Pull request by fbossen**
Partially addresses #224

### CODE REVIEW

**Review by randomPoison — APPROVED** [review]

File: meson.build [hunk 1]
```diff
@@ -30,7 +30,7 @@ project('dav1d', ['c'],
                       'b_ndebug=if-release'],
     meson_version: '>= 0.49.0')
 
-dav1d_soname_version       = '6.8.0'
+dav1d_soname_version       = '6.9.0'
```
Comment (kkysen) [inline meson.build:33]: Do we need to update this somewhere?

**Comment by kkysen** [comment]
I think the code this changes has subtly changed a lot after my other PRs merged, so I think it should be re-reviewed after being rebased.

**Comment by fbossen** [comment]
> I think the code this changes has subtly changed a lot after my other PRs merged, so I think it should be re-reviewed after being rebased.

There have been quite a few changes since this was initially written. Definitely needs another round of reviews. I am using `DRav1d<Vec<Rav1dITUTT35>, Vec<Dav1dITUTT35>>` to store the ITU-T T.35 data but am not convinced it's the best approach.

**Comment by fbossen** [comment]
@kkysen Can you have another look at this when you have time? I am sure there are better ways of doing this. Also, some more recent commits in dav1d build on top of this, so it may be good to have this integrated in a not too distant future.

**Review by kkysen — COMMENTED** [review]
@fbossen, can I take over this and fix the conversion issues?  I know the conversion code very well.

**Comment by fbossen** [comment]
> @fbossen, can I take over this and fix the conversion issues? I know the conversion code very well.

Please go ahead

**Comment by kkysen** [comment]
Ugh, why did this get automatically closed?

**Pull request closed without merging**